torch/distributed/_tensor/examples/comm_mode_features_example.py

```diff
@@ -507,6 +507,38 @@ def test_transformer_operation_tracing(self, is_seq_parallel: bool = False) -> N
         print(comm_mode.generate_operation_tracing_table())
         comm_mode.log_operation_tracing_table_to_file()
 
+    def test_MLP_json_dump(self) -> None:
```

Comment:
for example snippet, I'm not sure if we should name it with `test_` prefix. Does other reviewer stand with this decision? If no, I suggest that we rename them with prefix `example_`.